### ISSUE TITLE
Adjust bottom left camera position in the viewport to not obstruct transforms information.

### ISSUE THREAD

**Issue by viksl** [anthonyec/godot_little_camera_preview#5]
In the Image 1 below I'd like to suggest to set the bottom left view for the camera up so when you dock the camera to the left side the information at the bottom left corner (visible in the Image 1 below) can be still viewed. The transforms info is particularly useful when rotating stuff around even with camera visible as well as moving objects around.

In the Image 2 you can see that there's a status box with information about the scene/current view which is also often useful which makes the right side not the best option if you want to view the camera, the transforms info and the box at the same time.
All this could be resolved by moving the left docking position several pixesl up.

Image 1:
![image](https://github.com/anthonyec/godot_little_camera_preview/assets/33526847/002a02a7-5ce5-48ac-9cd6-1dc358519066)
Image 2:
![image](https://github.com/anthonyec/godot_little_camera_preview/assets/33526847/540c94cc-5977-4d6b-beab-d5bd976ed5c6)

I hope me description makes sense, otherwise it's a really cool plugin ;).

**Comment by anthonyec** [comment]
Thanks for the feedback and input! 

It's a little tricky because currently it's aligned with the other buttons in the viewport. So increasing it's Y margin will make it look a bit misaligned. This is the current alignment:

<img width="933" alt="guidelines_2-min" src="https://github.com/anthonyec/godot_little_camera_preview/assets/1451668/6b226741-8338-4be7-b894-4745067fed7e">

But yeah, I see how this is annoying if you rely on both scene information and transform info at the same time. 

Instead I'm thinking maybe the preview can fit alongside other elements in the viewport, adjusting it's position only when the information is visible:

<img width="933" alt="aligned-min" src="https://github.com/anthonyec/godot_little_camera_preview/assets/1451668/9ed2afc8-8a49-48f9-bb15-157441ee19f6">

What do you think? Would this work with your setup?

**Comment by viksl** [comment]
That's certainly nice! Though my main port wasn't that much about the right side as it was about the left side (bottom left corner shows the current operations as in my picture 1) which people check more often  (you need to move precisely with snap somthing or rotate or scale, it always shows what you are doing in the that left corner ;)).

It's a great plugin and I can't wait for one day when we don't have to install plugins per project basis because this will be a staple for me, thanks!

**Comment by anthonyec** [comment]
Yeah without just changing the alignment, which will make it look wonky, this is hard to fix with how the plugin is currently implemented. 

At the moment the camera preview sits above all viewport UI in the editor node tree. Ideally I'd like the preview to be part of the viewport node tree, where the bottom-left operation info could be shown **above** the camera preview. 

For now, I'd recommend [editing the margins](https://github.com/anthonyec/godot_little_camera_preview/blob/main/addons/anthonyec.camera_preview/preview.gd#L27) in projects that use the addon. Or using the preview in the bottom-right corner once I merge [this fix](https://github.com/anthonyec/godot_little_camera_preview/pull/6).
